### PR TITLE
Clinic db

### PR DESCRIPTION
**Pull request by karam084**
**As per Microverse, I need to work solo for this week after my partner does not meet.**

**Project requirements**

- Take a look at the following database diagram for a clinic:
  clinic_diagram
- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+CREATE INDEX treatment_id_asc ON invoice_items(treatment_id ASC);
+
+CREATE TABLE medical_histories_treatment(
+  treatment_id INT,
+  CONSTRAINT treatment_id FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+  medical_history_id INT,
+  CONSTRAINT medical_history_id FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+  PRIMARY KEY (treatment_id, medical_history_id)
+);
```
Comment (uche-inyama) [inline schema_based_on_diagram.sql]: Kindly index the foreign keys on medical_histories_treatment table.
Also, on line 75, you **added treatment_id** and **medical_history_id** as primary keys after referencing them as foreign keys on line 72 and 74. 
#### A table can only have one primary key.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
### Hi @karam, 

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there

## Required Changes ♻️

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (karam084) [reply schema_based_on_diagram.sql]: Thank you

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @karam084 , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: Descriptive PR.
:heavy_check_mark: Well implemented suggested change.
### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**